src/main/scala/Column.scala

```diff
@@ -1134,6 +1161,92 @@ case class Column private (
     }
   }
 
+  def validateValue(value: Any): Array[ErrorWithoutContext] =
+    value match {
+      case numericValue: Number => {
+        validateNumericValue(numericValue)
+      }
+      case s: String        => Array[ErrorWithoutContext]()
+      case d: ZonedDateTime => Array[ErrorWithoutContext]()
```

Comment:
I presume you'll be filling these in later?

Reply:
Yes